nz.mega.MEGAsync.metainfo.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2020 Artem Polishchuk <ego.cordatus@gmail.com> -->
+<component type="desktop-application">
+  <id>nz.mega.MEGAsync</id>
+  <metadata_license>CC0-1.0</metadata_license>
+
+  <!-- MEGAsync is under a proprietary license, except the SDK which is BSD -->
+  <project_license>LicenseRef-proprietary</project_license>
+
+  <name>MEGAsync</name>
+  <provides>
```

Comment:
I don't think it's needed. Adding `launchable` is, though.

Reply:
Replaced with `launchable`. Should i also rename .desktop file with proper UUID.desktop and try to send upstream? Flatpak this could fix automatically but this could ruin other non-flatpak build i suppose. Not so big deal though.

Reply:
`rename-desktop-file` & co exists because people usually don't follow the specification. Ideally it should be fixed upstream.

Reply:
`• tag-missing           : <name> is not present`
https://flathub.org/builds/#/builders/48/builds/3873

Probably we should back name here?